website/docs/en/config/experiments.mdx

```diff
@@ -225,6 +225,53 @@ module.exports = {
 };
 ```
 
+## experiments.incremental
+
+<ApiMeta addedVersion="1.1.0-beta.0" />
+
+- **Type:** `undefined | boolean | Incremental`
+- **Default:** `undefined`
+
+```ts
+type Incremental = {
+  make?: boolean;
+  emitAssets?: boolean;
+  inferAsyncModules?: boolean;
+  providedExports?: boolean;
+  dependenciesDiagnostics?: boolean;
+  modulesHashes?: boolean;
+  modulesCodegen?: boolean;
+  modulesRuntimeRequirements?: boolean;
+  buildChunkGraph?: boolean;
+};
+```
+
+Whether to enable incremental rebuild to speed up the rebuild speed.
+
+`true` means enable incremental for all stages. `false` means disable incremental for all stages. Incremental can also be enabled only for specified partial stages:
+
+```ts
+module.exports = {
+  experiments: {
+    // enable incremental for all stages
+    incremental: true,
+    // disable incremental for all stages
+    incremental: false,
+    // enable only for specified partial stages
+    incremental: {
+      make: true,
+      emitAssets: true,
+    },
+  },
+};
+```
+
+Currently, incremental for the make and emitAssets stages is enabled by default. This is also the default behavior since Rspack v1.0. As this feature further stabilizes, we will enable incremental for more stages by default.
```

Comment:
```suggestion
Currently, incremental for the `make` and `emitAssets` stages is enabled by default. This is also the default behavior since Rspack v1.0. As this feature further stabilizes, we will enable incremental for more stages by default.
```